go/hcti/main.go

```diff
@@ -0,0 +1,74 @@
+package hcti
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"time"
+)
+
+type Image struct {
+	Url string
+}
+
+type ImageError struct {
+	Error   string
+	Message string
+}
+
+func GenerateImage(html string, css string) (url string, elaspedMS int64, err error) {
+	userID := os.Getenv("API_ID")
+	apiKey := os.Getenv("API_KEY")
+
+	data := map[string]string{
+		"html": html,
+		"css":  css,
+	}
+	reqBody, err := json.Marshal(data)
+
+	if err != nil {
+		return "", 0, err
+	}
+
+	req, err := http.NewRequest("POST", "https://hcti.io/v1/image", bytes.NewReader(reqBody))
```

Comment:
If you use `encode` above, this can be : 
```suggestion
	req, err := http.NewRequest("POST", "https://hcti.io/v1/image", b)
```